### PR TITLE
Update create-a-task-sequence-to-upgrade-an-operating-system.md

### PR DESCRIPTION
**Pull request by raghuke**
### Summarize the change in the pull request title

Describe your change, specifically *why* you think it's needed.

Fixes #Issue_Number (if necessary)

### CODE REVIEW

**Comment by PRMerger9** [comment]
@raghuke : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@jborsecnik : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@jborsecnik : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/osd/deploy-use/create-a-task-sequence-to-upgrade-an-operating-system.md [hunk 1]
```diff
@@ -144,6 +144,15 @@ Starting in version 1802, the default task sequence template for Windows 10 in-p
 
 - **Network/wired connection checks**: Add steps in this group to check whether the computer is connected to a network, and isn't using a wireless connection. This action requires a custom script or utility to perform this check.  
 
+- **USB drive checks**: Add steps in this group to check whether a USB drive is connected to the computer. This action requires a custom script or utility. For example: Using WbemTest, connect to the `root\cimv2` namespace. Then run the following query: `Select * From Win32_Diskdrive where InterfaceType = "USB"`. If it returns any results, a USB drive is connected to the computer. Otherwise, no USB drive is connectd.  
+
+- **Device OS version, model, and manufacturer information**: Add steps in this group to check the OS version, model, and manufacturer of the computer. This action requires a custom script or utility to perform this check. For example: Using WbemTest, connect to the `root\cimv2` namespace. Then run the following queries: `Select model, manufacturer from Win32_Computersystem` and `Select Version From Win32_OperatingSystem`.
```
Comment (aczechowski) [inline sccm/osd/deploy-use/create-a-task-sequence-to-upgrade-an-operating-system.md]: Make and model are variables that also come with the [Set Dynamic Variables](https://docs.microsoft.com/sccm/osd/understand/task-sequence-steps#BKMK_SetDynamicVariables) step. So they can use a custom script or utility, but it's far easier to just add that step. 

I don't think we have any TS variable for OS version, but we should double-check with engineering. But I know you can use conditions based on it. So want to know more about why we should add OS version here?

File: sccm/osd/deploy-use/create-a-task-sequence-to-upgrade-an-operating-system.md [hunk 2]
```diff
@@ -144,6 +144,15 @@ Starting in version 1802, the default task sequence template for Windows 10 in-p
...
+
+- **Device OS version, model, and manufacturer information**: Add steps in this group to check the OS version, model, and manufacturer of the computer. This action requires a custom script or utility to perform this check. For example: Using WbemTest, connect to the `root\cimv2` namespace. Then run the following queries: `Select model, manufacturer from Win32_Computersystem` and `Select Version From Win32_OperatingSystem`. 
+
+- **Set power plan to "High Performance"**: Windows 10 has the pre-defined power plans "Balanced," "Power saver," and "High performance" to optimize the power usage of your device. If you're looking to boost performance, you can change to the "High performance" option, which speeds the upgrade process. For example: Add a **Run Command Line** to change the current power plan on the device:
```
Comment (aczechowski) [inline sccm/osd/deploy-use/create-a-task-sequence-to-upgrade-an-operating-system.md]: We need to make a comment here about resetting this at the end of the task sequence. What's the GUID to set the power plan back to Balanced? Perhaps also need to make a warning about its use, like they should check with their hardware manufacturer and test to make sure specific device models can handle this setting.

File: sccm/osd/deploy-use/create-a-task-sequence-to-upgrade-an-operating-system.md [hunk 3]
```diff
@@ -144,6 +144,15 @@ Starting in version 1802, the default task sequence template for Windows 10 in-p
...
+
+- **Device OS version, model, and manufacturer information**: Add steps in this group to check the OS version, model, and manufacturer of the computer. This action requires a custom script or utility to perform this check. For example: Using WbemTest, connect to the `root\cimv2` namespace. Then run the following queries: `Select model, manufacturer from Win32_Computersystem` and `Select Version From Win32_OperatingSystem`. 
+
+- **Set power plan to "High Performance"**: Windows 10 has the pre-defined power plans "Balanced," "Power saver," and "High performance" to optimize the power usage of your device. If you're looking to boost performance, you can change to the "High performance" option, which speeds the upgrade process. For example: Add a **Run Command Line** to change the current power plan on the device:
+</br>`PowerCfg.exe /s 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c</br>`
+
+- **Remove System Center Endpoint Protection (SCEP)**: Add steps in this group to remove SCEP with this version of Windows 10. The method to uninstall an application varies. For example, add a **Run Command Line**: </br>`C:\Windows\ccmsetup\scepinstall.exe /u /s`</br>
```
Comment (aczechowski) [inline sccm/osd/deploy-use/create-a-task-sequence-to-upgrade-an-operating-system.md]: Let's discuss and/or investigate this further - I thought Windows automatically removes SCEP and activates Defender.

**Comment by aczechowski** [comment]
After offline discussion with Raghu, we decided to close this.

**Pull request closed without merging**